db/migrate/20161015070842_setup_njdb.rb

```diff
@@ -13,21 +13,21 @@ def change
       p.string :name
```

Comment:
yeah definitely. keeps track of db schema history. in case we need to rollback a migration.